site/homepage/content/getting-started/index.md

```diff
@@ -121,46 +121,50 @@ Before a device can connect to Hono and publish any data, the corresponding info
 
 ### Creating a new Tenant
 
-Choose a (random) tenant identifier and register it using the Device Registry's HTTP API (replace `my-tenant` with your identifier):
+Register a tenant it using the Device Registry's management HTTP API (a random tenant identifier will be generated):
 ~~~sh
-export MY_TENANT=my-tenant
-curl -i -H "content-type: application/json" --data-binary '{"tenant-id": "'$MY_TENANT'"}' http://$REGISTRY_IP:28080/tenant
+curl -i -H "content-type: application/json" --data-binary '{}' http://$REGISTRY_IP:28080/tenants
 
 HTTP/1.1 201 Created
-Location:  /tenant/my-tenant
-Content-Length: 0
-~~~
+etag: resource_version_value
+Location:  /tenants/generated-tenant-id
 
-{{% note title="Conflict" %}}
-You will receive a response with a `409` status code if a tenant with the given identifier already exists.
-In this case, simply pick another (random) identifier and try again.
-{{% /note %}}
+{
+"id":"generated-tenant-id"
+}
+
+export MY_TENANT=generated-tenant-id
+~~~
 
 ### Adding a Device to the Tenant
 
-Choose a (random) device identifier and register it using the Device Registry's HTTP API (replace `my-device` with your identifier):
+Register a device using the Device Registry's management HTTP API (a random device identifier will be assigned):
 ~~~sh
-export MY_DEVICE=my-device
-curl -i -H "content-type: application/json" --data-binary '{"device-id": "'$MY_DEVICE'"}' http://$REGISTRY_IP:28080/registration/$MY_TENANT
+curl -i -H "content-type: application/json" --data-binary '{}' http://$REGISTRY_IP:28080/devices/$MY_TENANT
 
 HTTP/1.1 201 Created
-Location: /registration/my-tenant/my-device
-Content-Length: 0
+etag: resource_version_value
+Location: /devices/generated-tenant-id/generated-device-id
+
+{
+    "id": "generated-device-id"
+}
+
+export MY_DEVICE=generated-device-id
 ~~~
 
 ### Setting a Password for the Device
 
-Choose a (random) password and register it using the Device Registry's HTTP API (replace `my-pwd` with your password):
+Choose a (random) password and register it using the Device Registry's management HTTP API (replace `my-pwd` with your password):
 ~~~sh
 export MY_PWD=my-pwd
-curl -i -H "content-type: application/json" --data-binary '{
-  "device-id": "'$MY_DEVICE'",
+curl -i -X PUT -H "content-type: application/json" --data-binary '[{
   "type": "hashed-password",
   "auth-id": "'$MY_DEVICE'",
   "secrets": [{
       "pwd-plain": "'$MY_PWD'"
   }]
-}' http://$REGISTRY_IP:28080/credentials/$MY_TENANT
+}]' http://$REGISTRY_IP:28080/credentials/$MY_TENANT/$MY_DEVICE
```

Comment:
the response to this is a 204 (not a 201) if I am not mistaken ...

Reply:
you are correct !